pkg/transportrequest/gitutils.go

```diff
@@ -2,12 +2,73 @@ package transportrequest
 
 import (
 	"fmt"
+	gitUtils "github.com/SAP/jenkins-library/pkg/git"
+	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"github.com/go-git/go-git/v5"
 	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/pkg/errors"
+	"os"
 	"regexp"
 	"sort"
 )
 
+var logRange = gitUtils.LogRange
+var findLabelsInCommits = FindLabelsInCommits
+
+type iTransportRequestGitUtils interface {
```

Comment:
Up to now we have to my knowledge not the habit of marking interfaces with a leading "i".

Reply:
Well, i like it and I found it here: gitopsUpdateDeployment.go - see iGitopsUpdateDeploymentGitUtils.  diversity :-)